gradle/dependencies.gradle

```diff
@@ -54,7 +54,7 @@ libraries.fastutil =            [coordinates: 'it.unimi.dsi:fastutil', version:
 libraries.gcs =                 [coordinates: 'com.google.apis:google-api-services-storage', version: 'v1-rev136-1.25.0']
 libraries.groovy =              [coordinates: 'org.gradle.groovy:groovy-all', version: gradleGroovyVersion, because: 'emulating the Groovy 2.4-style groovy-all.jar, see https://github.com/gradle/gradle-groovy-all']
 libraries.gson =                [coordinates: 'com.google.code.gson:gson', version: '2.8.5']
-libraries.guava =               [coordinates: 'com.google.guava:guava', version: '26.0-android', because: 'android variant is compatible with JDK 7']
+libraries.guava =               [coordinates: 'com.google.guava:guava', version: '27.1-android', because: 'android variant is compatible with JDK 7']
```

Comment:
The `because` is not true anymore. It should be `android variant is faster because of https://github.com/google/guava/issues/3223`.